net/remote/remote-client.sh

```diff
@@ -38,6 +38,10 @@ local|tar)
   PATH="$HOME"/.cargo/bin:"$PATH"
   export USE_INSTALL=1
   export SOLANA_DEFAULT_METRICS_RATE=1
```

Comment:
After adding below line, the testnet is up and running
`./fetch-perf-libs.sh
`